examples/logger.py

```diff
@@ -3,20 +3,60 @@
 """
 
 import time
-from typing import Any, Optional, Tuple
+import logging
+from typing import Any, Optional, Tuple, Callable
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+
```

Comment:
Please remove the empty line.
We usually make only one line after imporing modules.